addons/local-path-provisioner/template/testgrid/k8s-docker.yaml

```diff
@@ -57,3 +65,11 @@
     localPathProvisioner:
       version: "__testver__"
       s3Override: "__testdist__"
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_read_write_object_store rwtest testfile.txt
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_testfile rwtest testfile.txt
```

Comment:
does this validate write after upgrade? any reason not to add validate_read_write_object_store after this?

Reply:
good idea, I've added that now